src/validators/account-validator.js

```diff
@@ -0,0 +1,231 @@
+        .isLength({ min: 8, max:254 }).trim().withMessage('Password must have a minimum of 8 characters and a maximum of 254 characters')
+]
+
+
+const validationLogin = [
+    validationUsername,
+    validationPassword,
+    (request, response, next) => {
+        validateResult(request, response, next);
+    }
+]
+
+const validationConfirmationAccount = [
+    validationUsername,
+    check('codeConfirmation')
+        .exists().withMessage('El campo debe existir')
+        .notEmpty().withMessage('El campo no debe estar vacío')
+        .not().isString().withMessage('El campo debe ser un número')
+        .matches(/^[0-9]{6}$/).withMessage('Permitir solo números del 0 al 9 y deben ser 6 dígitos'),
+    (request, response, next) => {
+        validateResult(request, response, next);
+    }
+]
+
+const validationSendEmail = [
+    validationEmail,
+    (request, response, next) => {
+        validateResult(request, response, next);
+    }
+]
+
+const validationAccountEmail = [
+    validationEmail,
+    check('title')
+        .notEmpty().withMessage('El campo no debe estar vacío')
+        .isString().withMessage('El campo debe ser una cadena')
+        .isLength({ min: 5, max: 100}).withMessage('El titulo del mensaje debe tener un mínimo de 5 caracteres y un máximo de 100 caracteres')
+        .matches(/^[a-zA-ZÑñÁáÉéÍíÓóÚúÜü0-9.,#]+(\s*[a-zA-ZÑñÁáÉéÍíÓóÚúÜü0-9.,#]*){5,600}/).withMessage('Solo letras de la A a la Z, números del 0 al 9, caracteres., # y espacios'), 
+    check('message')
+        .notEmpty().withMessage('El campo no debe estar vacío')
+        .isString().withMessage('El campo debe ser una cadena')
+        .isLength({ min: 5, max: 600}).withMessage('El mensaje debe tener un mínimo de 5 caracteres y un máximo de 600 caracteres')
+        .matches(/^[a-zA-ZÑñÁáÉéÍíÓóÚúÜü0-9.,#]+(\s*[a-zA-ZÑñÁáÉéÍíÓóÚúÜü0-9.,#]*){5,600}/).withMessage('Solo letras de la A a la Z, números del 0 al 9, caracteres., # y espacios'), 
+    (request, response, next) => {
+        validateResult(request, response, next);
+    }
+]
+
+const validationStatusAccount  = [
+    check('status')
+        .exists().withMessage('El campo debe existir')
+        .notEmpty().withMessage('El campo no debe estar vacío')
+        .isNumeric().withMessage('El campo no debe ser un número')
+        .matches(/^[1-3]$/).withMessage('Permitir solo números del 1 al 3')
+]
+
+
+const validationCreationAccount = [
+    validationAccount,
+    validationUsername,
+    validationEmail,
+    validationPassword,
+    check('role')
+        .exists().withMessage('El campo debe existir')
+        .isString().withMessage('El campo debe ser una cadena')
+        .isIn(['user', 'manager']).withMessage('El rol no es válido'),
+    check('idCity')
+        .exists().withMessage('El campo debe existir')
+        .notEmpty().withMessage('El campo no debe estar vacío')
+        .matches(/^[a-z0-9]{24}$/).withMessage('El ID debe tener números y letras minúsculas')
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Martha-Miroslava/APIBiziidiDiidxaza/pullRequest?prid=8267260)
```suggestion
        .matches(/^[a-z0-9]{24}$/).withMessage("El ID debe tener números y letras minúsculas")
```